src/Common/src/CoreLib/Interop/Unix/System.Native/Interop.GetTimestamp.cs

```diff
@@ -8,10 +8,10 @@ internal static partial class Interop
 {
     internal static partial class Sys
     {
-        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetTimestampResolution")]
-        internal static extern bool GetTimestampResolution(out long resolution);
+        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetTimestampResolution", ExactSpelling = true)]
+        internal static extern long GetTimestampResolution();
```

Comment:
Nit: The actual PAL return value is ulong.

Reply:
Fixing the native signatures to return `int64_t` seems the most correct. We need to return `long` (ultimately) since the public `Stopwatch.GetTimestamp` method returns `long`. However, their is a potential (but incredibly unlikely) concern that the Unix implementation could overflow since OSX actually returns `uint64_t` and since the POSIX spec doesn't specify the sign of `time_t`.